src/pyedb/dotnet/edb.py

```diff
@@ -510,6 +511,11 @@ def sources(self):
         terms = [term for term in self.layout.terminals if int(term.GetBoundaryType()) in [3, 4, 7]]
         return {ter.GetName(): ExcitationSources(self, ter) for ter in terms}
 
+    @property
+    def voltage_regulator_modules(self):
+        """Get all voltage regulator modules"""
+        return self._vrms
+
     @property
     def probes(self):
```

Comment:
You can get all voltage regulators from edb from Layout.VoltageRegulators. There is no need to cache it.

Reply:
ah let me try

Reply:
fixed